site/app/templates/officeHoursQueue/QueueFilter.twig

```diff
@@ -10,7 +12,21 @@
     <a id="new_queue_rand_token" onclick="genRandCode('new_queue_token')" onkeypress="genRandCode('new_queue_token')" title="Generate random access code" aria-label="Generate random access code" tabindex="0" style="padding:.5rem;">
       <i class="fas fa-sync"></i>
     </a>
-    <button id="open_new_queue_btn" type="submit" class="btn btn-primary">Open New Queue</button>
+      {% if viewer.isContactInfoEnabled() %}
+        <input type="text" id="regex" name="regex" placeholder="Contact Info Regex Pattern" aria-label="Regex" style="min-width: 33%;">
+
+        <div class="regex-help">
+            <br>You may optionally specify a regular expression to ensure your students enter the requested
+            format for the contact information.  For example: <br>
+            <b>Gmail:&nbsp;&nbsp;</b> <tt>.+@gmail.com</tt><br>
+            <b>Email:&nbsp;&nbsp;</b> <tt>.+@.+\..+</tt><br>
+            <b>Webex Personal Room:&nbsp;&nbsp;</b> <tt>^https:\/\/.+\.webex\.com\/meet\/.*</tt><br>
+            <b>Webex Meeting:&nbsp;&nbsp;</b> <tt>^https:\/\/.+\.webex\.com\/.*</tt><br>
+            <b>Skype link:&nbsp;&nbsp;</b> <tt>.*join.skype.com.*</tt><br>
+            <b>Zoom link:&nbsp;&nbsp;</b> <tt>.*zoom.us.*</tt><br><br>
```

Comment:
```suggestion
            <b>Skype link:&nbsp;&nbsp;</b> <tt>.*join\.skype\.com.*</tt><br>
            <b>Zoom link:&nbsp;&nbsp;</b> <tt>.*zoom\.us.*</tt><br><br>
```
These periods need to be escaped otherwise they mean any character.

Reply:
thanks for catching that